src/extensions/default/Inline3DParametersEditor/ParameterEditor.js

```diff
@@ -113,7 +112,7 @@ define(function (require, exports, module) {
         this._hideInputs();
 
         this._values = [];
-        for(var i = 0; i<this._numberOfParameters; i++) {
+        for(var i = 0; i < this._numberOfParameters; i++) {
```

Comment:
Yes, this can be replaced by the same.
But the number of parameters in this.$sliders remains constant (i.e. 3). We only hide those in case the number of parameters are less than 3. 
If we use the above code snippet, the size of this._values will always be 3. 
Hence the for loop. It restricts the length of this._values to the number of parameters.